hummingbot/model/funding_payment.py

```diff
@@ -73,7 +63,7 @@ def to_pandas(cls, payments: List):
             index += 1
             data.append([
                 index,
-                datetime.fromtimestamp(int(payment.timestamp / 1e3)).strftime("%Y-%m-%d %H:%M:%S"),
+                datetime.fromtimestamp(int(float(payment.timestamp) / 1e3)).strftime("%Y-%m-%d %H:%M:%S"),
```

Comment:
is necessary to cast to float to make the division? was a Decimal before?

Reply:
I think it failed the tests. It could that it was expected to be `payment.timestamp / Decimal("1e3")`